### PR TITLE
Create Windows Server build from ISO wf and doc

### PR DESCRIPTION
**Pull request by rofuentes**
This PR includes:
- Bootstrap and Post-Install PowerShell files
- Windows Build base workflow
- Windows Server 2016 workflow
- Brief documentation describing workflow configuration and invocation

### CODE REVIEW

File: daisy_workflows/image_build/windows/components/SetupComplete.cmd [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+if not exist D:\ (
+  (echo select disk 1 & echo online disk & echo attribute disk clear readonly) | diskpart
+) > COM1:
+echo. > COM1:
+
+REM This sleep fixes a multitude of timing problems
+REM We use ping instead of timeout as timeout seems to have timing
+REM problems if the clock resets (as it does on first boot)
+ping 127.0.0.1 -n 60
```
Comment (adjackura) [inline daisy_workflows/image_build/windows/components/SetupComplete.cmd]: You can remove this now

File: daisy_workflows/image_build/windows/README.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
+### Windows ISO to Google Compute Image Build
```
Comment (adjackura) [inline daisy_workflows/image_build/windows/README.md]: Can you call out that this is just 2016

File: daisy_workflows/image_build/windows/components/SetupComplete.cmd (hunk 1 above)
Comment (rofuentes) [reply daisy_workflows/image_build/windows/components/SetupComplete.cmd]: Done.

File: daisy_workflows/image_build/windows/README.md (hunk 2 above)
Comment (rofuentes) [reply daisy_workflows/image_build/windows/README.md]: I added a table detailing the supported WS versions and recommended updates to include.

**Pull request merged**